maps-app-ios/Maps App/App Notifications/LoginNotifications.swift

```diff
@@ -40,12 +40,12 @@ extension MapsAppNotifications {
 
 // MARK: Internals
 extension MapsAppNotifications {
-    static func postLoginNotification(user:AGSPortalUser) {
+    static func postSignInNotification(user:AGSPortalUser) {
         NotificationCenter.default.post(name: MapsAppNotifications.Names.appLogin, object: mapsApp,
```

Comment:
Should we also update the name of the notification definition to `MapsAppNotifications.Names.appSignIn`?